src/index.js

```diff
@@ -1,2 +1,221 @@
-import './style.css';
-console.log('hello')
\ No newline at end of file
+import "./style.css";
+const DisplayItems=document.querySelector('.DisplayItems')
+const GetItems=localStorage.getItem('Todo')
+const DeleteItem=document.querySelectorAll('.RemoveItems')
+// const DeleteItems=document.querySelector('.')
+const Items=document.querySelector('.NewItem')
+let ItemsContainer=[]
+let holdItems={}
+
+const Validate = () => {
+if(Items.value !==''){
+   return true;
+}else{
+   return false;
+}
+}
+
+const ShowAllItems = () =>{
+   DisplayItems.innerHTML='';
+   if(GetItems !== null){
+    const AllItems=JSON.parse(localStorage.getItem('Todo'))
+    ItemsContainer=AllItems;
+    AllItems.forEach((item,index) => {
+       DisplayItems.innerHTML+= `
+       <div class="TodoList">
+       <div class="form">
+    <input type="checkbox" placeholder="..." class="cancelItem" id="${index}"> 
+    <input type="text" class="list-items" value="${item.Description}" id="${index}">
+   </div>
+   <span class="RemoveItems" button id="${index}"><i  id="del+${index}" class="fa fa-trash" class="del" name="del" aria-hidden="true"></i><i id="edit+${index}" class="fas fa-ellipsis-v"></i></span>
+   </div>
+           `  
+    });
+   }else{
+       []
+   }
+}
+
+const ShowItems = () => {
+   window.location.reload();
+   DisplayItems.innerHTML='';
+   const AllItems=JSON.parse(localStorage.getItem('Todo'))
+   ItemsContainer=AllItems;
+   ItemsContainer.forEach((item,index) => {
+   DisplayItems.innerHTML+= `
+   <div class="TodoList">
+   <div class="form">
+<input type="checkbox" placeholder="..." class="cancelItem" id="${index}"> 
+   <input type="text" class="list-items" value="${item.Description}" id="${index}">
+</div>
+   <span class="RemoveItems" button id="${index}"><i  id="del+${index}" class="fa fa-trash" aria-hidden="true"></i><i id="edit+${index}" class="fas fa-ellipsis-v"></i></span>
+</div>` 
```

Comment:
- [ ] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

![image](https://user-images.githubusercontent.com/81932478/193137782-75b87faa-f893-445f-a2fe-f4741feab5a3.png)

- This means the first value of the index of the entry in your array should be 1 not 0, and the second index should be `2` not `1`, kindly fix this issue.